spec/index.bs

```diff
@@ -2785,18 +2785,50 @@ interface GPUTexture {
     GPUTextureView createView(optional GPUTextureViewDescriptor descriptor = {});
 
     undefined destroy();
+
+    [SameObject] readonly attribute GPUExtent3DDict size;
```

Comment:
```suggestion
    readonly attribute GPUExtent3DDict size;
```

> The [[SameObject](https://webidl.spec.whatwg.org/#SameObject)] extended attribute must not be used on anything other than a [read only](https://webidl.spec.whatwg.org/#dfn-read-only) [attribute](https://webidl.spec.whatwg.org/#dfn-attribute) whose type is an [interface type](https://webidl.spec.whatwg.org/#idl-interface) or [object](https://webidl.spec.whatwg.org/#idl-object).

Dictionaries get converted into a new JS value each time since they don't have corresponding JS objects like interface objects do.